cluster-autoscaler/simulator/drain.go

```diff
@@ -35,8 +35,8 @@ import (
 // along with their pods (no abandoned pods with dangling created-by annotation). Useful for fast
 // checks.
 func FastGetPodsToMove(nodeInfo *schedulernodeinfo.NodeInfo, skipNodesWithSystemPods bool, skipNodesWithLocalStorage bool,
-	pdbs []*policyv1.PodDisruptionBudget) ([]*apiv1.Pod, error) {
-	pods, err := drain.GetPodsForDeletionOnNodeDrain(
+	pdbs []*policyv1.PodDisruptionBudget) ([]*apiv1.Pod, *drain.BlockingPod, error) {
```

Comment:
Not in scope of this PR, but it seems like blocking pod and error could be combined (and error messages lazily evaluated - we discard them in some cases).